src/strings/en-us.json

```
@@ -97,7 +97,7 @@
     "ButtonMore": "More",
     "ButtonNextTrack": "Next track",
     "ButtonOk": "Ok",
-    "ButtonOpen": "Open",
+    "ButtonEditUser": "Edit User Permissions",
```

```suggestion
    "ButtonEditUser": "Edit user",
```

The key says nothing about permissions, also do not use word casing I hate it.